container-runtime/Makefile

```diff
@@ -37,11 +37,11 @@ include ${TOP}/make/actions.mk
 
 # Building crun shows errors indicating generated header files are not built yet,
 # so force single threading the make.
-all:	$(CRUNDIR)/Makefile .makefile_check | ${KM_OPT_BIN_PATH}
+all:	$(CRUNDIR)/Makefile .makefile_check | ${KM_OPT_BIN}
 	make -j1 -C $(CRUNDIR) all
-	cp $(CRUNDIR)/crun ${KM_OPT_BIN_PATH}
-	ln -f ${KM_OPT_BIN_PATH}/crun ${KM_OPT_BIN_PATH}/krun
-	ln -f ${KM_OPT_BIN_PATH}/crun ${KM_OPT_BIN_PATH}/krun-label-trigger
+	cp $(CRUNDIR)/crun ${KM_OPT_BIN}
+	ln -f ${KM_OPT_BIN}/crun ${KM_OPT_BIN}/krun
+	ln -f ${KM_OPT_BIN}/crun ${KM_OPT_BIN}/krun-label-trigger
 	mkdir -p ${BLDDIR}
```

Comment:
Why are this changes in this PR? crun/krun isn't part of the coverage, AFAIK

Reply:
They should not be and they are in master - might me my screwup from before?